_posts/2021-01-07-cockpit-235.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Cockpit 235
+author: pitti
+date: 2021-01-07
+tags: cockpit linux
+slug: cockpit-235
+category: release
+summary: Cockpit with SSH key improvements and motd editing
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 235.
+
+## Login: Improved handling of SSH host keys
+
+When connecting to a secondary host directly from the login page, unknown and changed host keys are handled in a better way. The situation is explained and accepted keys are remembered in the Browser:
```

Comment:
```suggestion
This release improves key management while logging into secondary hosts. The log in screen now explains unknown and altered keys&mdash;and also how to verify these new keys. Browsers remember accepted keys in client-side storage to improve the user experience.
```

1. Rearranged a lot of text to make it active voice and simplify the text.
2. "Browser" is not a proper noun, so it should be "browser". However, I rewrote it and started the sentence with "browser", so it's capital anyway in my rewrite… but for a different reason. :wink:
3. Clarified that this is in browser storage.
4. There shouldn't be a colon here.
5. Used an [em dash](https://www.lexico.com/grammar/dash-and-em-dash) (`&mdash;` in HTML (and Markdown), which looks like —) to connect a related sentence fragment to the previous sentence. I did this to offset the "and", so it doesn't look like an ambiguous "x and y and x". Without this, it _could_ look like "x and (y and z)" instead of "(x and y) and z".